README.md

```diff
@@ -115,12 +115,15 @@ This will allow you to build the *exact* same TCs used to build the nightlies, t
 
 On Mac OS X you may need to install the following tools using [Homebrew](https://brew.sh/):
 ```
-brew install nasm binutils libtool autoconf automake cmake makedepend sdl2 lua51 gettext pkg-config wget md5sha1sum
+brew install nasm binutils libtool autoconf automake cmake makedepend sdl2 lua@5.1 luarocks gettext pkg-config wget md5sha1sum
 echo 'export PATH="/usr/local/opt/gettext/bin:$PATH"' >> "$HOME"/.bash_profile
 ```
 
 If you run into a gettext error while building glib, try `brew link --force gettext` to override the built-in Mac OS BSD gettext with GNU GetText.
 
+*Note:* in Mojave you need to set a minimum deployment version higher than 10.04. Otherwise you'll get the error `ld: library not found for -lgcc_s.10.4`
```

Comment:
```suggestion
*Note:* in Mojave (10.14) you need to set a minimum deployment version higher than 10.04. Otherwise you'll get the error `ld: library not found for -lgcc_s.10.4`.
```

Although I've recently made up a couple of cutesy release names for KOReader, too often I forget what those things actually refer to.

As an aside, I'm slightly confused by some of the numbering here. It went straight from 10.04 to 10.10 or something?

Reply:
> As an aside, I'm slightly confused by some of the numbering here. It went straight from 10.04 to 10.10 or something?

10.06 should work too (and any other number in between). I just set 10.10 because it is the minimum osx release supported by brew. If we redistribute an app it does make sense targetting the oldest mac release (for reference official python binaries are using 10.06).

I can test 10.06 if you wish :+1:

Reply:
@Frenzie 

I get a bunch of warnings:

```
clang: warning: libstdc++ is deprecated; move to libc++ with a minimum deployment target of OS X 10.9 [-Wdeprecated]
```

and the build fails.

MACOSX_DEPLOYMENT_TARGET=10.09 works fine.

Reply:
Did you try adding `-stdlib=libc++` to the compiler flags?

But yeah, otherwise 10.09 it is.

Reply:
I think that modifying the environment is better(I didn't try your suggestion yet :d). Most people building on mojave have XCode10.app (the full blown sdk for developing "polite" applications for their systems). 

XCode will follow Apple rules, of course. https://developer.apple.com/documentation/xcode_release_notes/xcode_10_release_notes

Look at Deprecation Notices.

Reply:
Just to add my two cents about not trusting the DEPLOYMENT_TARGET too much: there's a very high probability something will pickup a symbol that wasn't actually supported on earlier macOS versions, despite the DEPLOYMENT_TARGET.

That's from personal experience with KindleTool, and some autotools projects picking up fstatat/openat, which were introduced in 10.10, and that despite a `MACOSX_DEPLOYMENT_TARGET=10.6` & `-mmacosx-version-min=10.6`...

Reply:
@NiLuJe: good to know, thanks.

I would love to build an application for mac as part of nightlies (like the appimage) but there is no way we can end up with a docker image tailored for macos. Even when [cross toolchains](https://github.com/tpoechtrager/osxcross) exist and work they rely on apple sdk, which AFAIK can't be redistributed freely.

Reply:
I was going to ask if perhaps it's legal to do [this](https://github.com/koreader/koreader-base/blob/66aa4585de16f289f5476ce758f001ce507041a3/Makefile#L178-L185), but…

![screenshot_2018-12-11_07-51-57](https://user-images.githubusercontent.com/202757/49783328-c40d0f00-fd19-11e8-9a1f-81b893c1b57f.png)

Reply:
@Frenzie: yep, the only legal alternative that I've found is https://www.macstadium.com/opensource